### ISSUE TITLE
TypeError: '<' not supported between instances of 'NoneType' and 'int'

### ISSUE THREAD

**Issue by arthurlutz** [mopidy/mopidy-local#36]
```
ERROR    [MainThread] mopidy.__main__ '<' not supported between instances of 'NoneType' and 'int'
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/mopidy/__main__.py", line 143, in main
    return args.command.run(args, proxied_config)
  File "/usr/local/lib/python3.7/dist-packages/mopidy_local/commands.py", line 102, in run
    limit=args.limit,
  File "/usr/local/lib/python3.7/dist-packages/mopidy_local/commands.py", line 224, in _scan_metadata
    elif result.duration < MIN_DURATION_MS:
TypeError: '<' not supported between instances of 'NoneType' and 'int'
Traceback (most recent call last):
  File "/usr/bin/mopidy", line 11, in <module>
    load_entry_point('Mopidy==3.0.1', 'console_scripts', 'mopidy')()
  File "/usr/lib/python3/dist-packages/mopidy/__main__.py", line 143, in main
    return args.command.run(args, proxied_config)
  File "/usr/local/lib/python3.7/dist-packages/mopidy_local/commands.py", line 102, in run
    limit=args.limit,
  File "/usr/local/lib/python3.7/dist-packages/mopidy_local/commands.py", line 224, in _scan_metadata
    elif result.duration < MIN_DURATION_MS:
TypeError: '<' not supported between instances of 'NoneType' and 'int'
```

```
# python3 -m pip list | grep Local
Mopidy-Local              3.1.0         
```

**Comment by arthurlutz** [comment]
Fixed by #35 when it will be integration.

**Comment by jodal** [comment]
Fixed by #35.
